data/quotes.json

```diff
@@ -701,4 +701,476 @@
+            "year": ""
+        },
+        {
+            "quote": "I've made a huge mistake.",
+            "character": "",
+            "actor": "",
+            "movie": "Arrested Development",
+            "year": ""
+        },
+        {
+            "quote": "What'chu talkin' 'bout, Willis?",
+            "character": "",
+            "actor": "",
+            "movie": "Different Strokes",
+            "year": ""
+        },
+        {
+            "quote": "So pick me, choose me, love me.",
+            "character": "",
+            "actor": "",
+            "movie": "Grey's Anatomy",
+            "year": ""
+        },
+        {
+            "quote": "Treat. Yo. Self.",
+            "character": "",
+            "actor": "",
+            "movie": "Parks and Recreation",
+            "year": ""
+        },
+        {
+            "quote": "If I die, turn my tweets in to a book.",
+            "character": "",
+            "actor": "",
+            "movie": "Brooklyn Nine-Nine",
+            "year": ""
+        },
+        {
+            "quote": "Saving people, hunting things, the family business.",
+            "character": "",
+            "actor": "",
+            "movie": "Supernatural",
+            "year": ""
+        },
+        {
+            "quote": "Hello brother.",
+            "character": "",
+            "actor": "",
+            "movie": "The Vampire Diaries",
+            "year": ""
+        },
+        {
+            "quote": "Is that your final answer?",
+            "character": "",
+            "actor": "",
+            "movie": "Who Wants to Be a Millionaire?",
+            "year": ""
+        },
+        {
+            "quote": "Nobody asked you Patrice!",
+            "character": "",
+            "actor": "",
+            "movie": "How I Met Your Mother",
+            "year": ""
+        },
+        {
+            "quote": "How you doin'?",
+            "character": "",
+            "actor": "",
+            "movie": "Friends",
+            "year": ""
+        },
+        {
+            "quote": "Let's be careful out there.",
+            "character": "",
+            "actor": "",
+            "movie": "Hill Street Blues",
+            "year": ""
+        },
+        {
+            "quote": "I want to go to there.",
+            "character": "",
+            "actor": "",
+            "movie": "30 Rock",
+            "year": ""
+        },
+        {
+            "quote": "It's handled.",
+            "character": "",
+            "actor": "",
+            "movie": "Scandal",
+            "year": ""
+        },
+        {
+            "quote": "Forgive me, but to me you've been dead for centuries.",
+            "character": "",
+            "actor": "",
+            "movie": "The Flash",
+            "year": ""
+        },
+        {
+            "quote": "Oy with the poodles already.",
+            "character": "",
+            "actor": "",
+            "movie": "Gilmore Girls",
```

Comment:
Instead of using movie as a field, we can change this field name to "source" for both movies and shows, to keep things uniform